recipe/build_geant4.sh

```diff
@@ -38,7 +38,6 @@ cmake                                                          \
       -DBUILD_SHARED_LIBS=ON                                   \
       -DGEANT4_INSTALL_EXAMPLES=ON                             \
       -DGEANT4_INSTALL_DATA=OFF                                \
-      -DGEANT4_BUILD_MULTITHREADED=ON                          \
```

Comment:
In the interests of being explicit how about:
```bash
      -DGEANT4_BUILD_MULTITHREADED=OFF                         \
```

Reply:
Fixed!